react/components/OrganizationUsersTable.tsx

```diff
@@ -574,7 +574,20 @@ const OrganizationUsersTable: FunctionComponent<Props> = ({
     ) {
       // eslint-disable-next-line no-useless-return
       return
-    }
+    } else {
+      setEditUserDetails({
+      id: rowData.id,
+      roleId: rowData.roleId,
+      userId: rowData.userId,
+      clId: rowData.clId,
+      orgId: rowData.orgId,
+      costId: rowData.costId,
+      name: rowData.name,
+      email: rowData.email,
+      canImpersonate: rowData.canImpersonate,
+    })
+    setEditUserModalOpen(true)
+  }
```

Comment:
This is the same logic used in the first if statement a few lines above. Instead of repeating it here, this method can be rewritten in a simpler way like this:

```js
const handleRowClick = ({ rowData }: CellRendererProps) => {
  const canClick = canManageOrganization || ruleClickEnabled({
    isAdmin,
    canEditSales,
    slug: rowData.role?.slug ?? '',
    canEdit,
    isSalesAdmin,
  })
  if (!canClick) return
  
  setEditUserDetails({
    id: rowData.id,
    roleId: rowData.roleId,
    userId: rowData.userId,
    clId: rowData.clId,
    orgId: rowData.orgId,
    costId: rowData.costId,
    name: rowData.name,
    email: rowData.email,
    canImpersonate: rowData.canImpersonate,
  })
  setEditUserModalOpen(true)
}
```

That way the repetition can be avoided and the final code is easier to understand as well.

Reply:
Thanks, @mairatma 
Just pushed a commit with your suggestion 👍